src/views/Help/Section508/index.tsx

```diff
@@ -37,22 +39,7 @@ const PageContent = () => {
         })}
       </CardGroup>
       <Divider />
-      <HelpContacts type="Section 508" />
-    </>
-  );
-};
-
-const Section508 = () => {
-  return (
-    <>
-      <HelpBreadcrumb type="Back" />
-      <Switch>
-        <Route
-          path="/help/section-508/steps-involved"
-          render={() => <StepsInvolved />}
-        />
-        <PageContent />
-      </Switch>
+      <HelpContacts type="IT Governance" />
```

Comment:
Sorry for the late Change Request! Noticed that this is displaying the IT Governance contacts for the Section 508 page.